controllers/sr_controller/runtime.ini

```diff
@@ -0,0 +1,3 @@
+[environment variables with relative paths]
```

Comment:
This is a section heading in `ini` format I think? Or is this meant to be a comment? (I think comments in `ini` normally start with `;`?)

Reply:
Nope, this a section heading, just a _really_ weirdly named one!